cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java

```diff
@@ -88,10 +88,10 @@ private static void shutDownMySQLAbandonedConnectionCleanupThread(ClassLoader cl
       Class<?> mysqlCleanupThreadClass = classLoader.loadClass("com.mysql.jdbc.AbandonedConnectionCleanupThread");
       Method shutdownMethod = mysqlCleanupThreadClass.getMethod("shutdown");
       shutdownMethod.invoke(null);
-      LOG.info("Successfully shutdown MySQL connection cleanup thread.");
+      LOG.info("Successfully shutdown RDBMS connection cleanup thread.");
```

Comment:
yeah, I don't think we should do this. Other DBs do not have this thread to cleanup. Unfortunately there is no way to check if it is mysql that we're running with, without adding an explicit dependency on it. We should probably just suppress the stack trace and move it to loglevel ``trace``?